openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityLineage/CustomControls.component.tsx

```diff
@@ -131,6 +144,56 @@ const CustomControls: FC<ControlProps> = ({
     [lineageData]
   );
 
+  // ------------------
+
+  type EntityDetails = {
+    [key: string]: any;
+  };
+
+  const getNodesTags = async (nodes: SelectedNode[]) => {
+    const nodesTags: EntityTags[] = [];
+
+    for (const node of nodes) {
+      const entity = (await fetchEntityDetails(node)) as EntityDetails;
```

Comment:
@karanh37 
Do we have any better approach or a better function for this purpose?

Reply:
We are thinking to move lineage data to elastic search, which will then enable us to fetch results from there.
It will be then easier to manage this.
For now putting this PR on hold.